src/drivers/uavcan_v1/Uavcan.cpp

```diff
@@ -339,143 +339,17 @@ void UavcanNode::Run()
 			PX4_INFO("received Port ID: %d", receive.port_id);
 
 			if (receive.port_id == uavcan_pnp_NodeIDAllocationData_1_0_FIXED_PORT_ID_) {
-				uavcan_pnp_NodeIDAllocationData_1_0 msg;
```

Comment:
I was thinking about playing with using this PR with UAVCAN v1, but it quickly became clear that would be a larger undertaking... I can leave the UAVCANv1 commit to a separate PR